core/domain/story_domain_test.py

```diff
@@ -102,6 +102,19 @@ def test_get_acquired_skill_ids_for_node_ids(self):
             ['skill_1', 'skill_2']
         )
 
+    def test_get_acquired_skill_ids_for_node_ids_special_cases(self):
```

Comment:
I don't see a test for the [] case (no acquired skill IDs).

Reply:
Done.